CMakeLists.txt

```
@@ -13,8 +13,20 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 
-# TODO: re-enable /GL and /LTCG when we get a bigger build machine
-add_compile_options(/await /bigobj /GR- /permissive-)
+if ("MSVC")
```

If it is in quotes, doesn't that make it a string (which is always false)? Shouldn't it be unquoted to use the variable name instead?

```suggestion
if (MSVC)
```